configuration/pih/htmlforms/covid19Intake.xml

```diff
@@ -262,12 +262,11 @@
 
         </includeIf>
 
-        <!-- all users that don't have retroConsultNote privilege cannot edit provider, location or date when active visit -->
-        <includeIf velocityTest="(!$user.hasPrivilege('Task: emr.retroConsultNote')) and ($visit.open)">
```

Comment:
So while I was reviewing the three possible scenarios, I discovered a bug... if a user did not have either of the retro consult note privileges and the visit would not open, none of three encounter details elements would be used, and there's be a stack trace when saving the form.

This was because (I assume?) in our original design you never should have been able to reach that case... ie, you shouldn't have been able to add a retro note if you didn't have either of the retro privileges.  But we'd gotten away from this.